twitter/twitterapp/views.py

```diff
@@ -0,0 +1,18 @@
+from django.shortcuts import render
+from twitterapp.models import Tweet
+from django.http import HttpResponse
+
+# Create your views here.
+def all_tweets(request):
+    tweets = Tweet.objects.all()
+    output = ""
+    for tweet in tweets:
+        output += "[{}] @{} - {}\n".format(tweet.date,tweet.user.name,tweet.content)
```

Comment:
```suggestion
        output += "[{}] @{} - {}\n".format(tweet.date, tweet.user.name, tweet.content)
```

Add Empty space after comma.